normandy/control/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+from django.conf.urls import url, include
+
+from normandy.control import views as control_views
+from django.contrib.auth.views import login, logout_then_login
+
+urlpatterns = []
+
+if settings.ADMIN_ENABLED:
+  urlpatterns += [
+      url(r'^control/', include([
+          url(r'^$', control_views.IndexView.as_view(), name='index'),
```

Comment:
I putted around with namespaces when I was overthinking all the registration/auth stuff, and they seemed really useful. Except my impression was that you would use `control` as the namespace, and the url name would still just be `index`. Then the urls in your template would look like `{% url 'control:index' %}`.  I think the answer given here was pretty useful http://stackoverflow.com/questions/19171570/a-real-example-of-url-namespace  ?